spyder_kernels/utils/mpl.py

```diff
@@ -27,11 +20,11 @@
 
 # Mapping of matlotlib backends options to Spyder
 MPL_BACKENDS_TO_SPYDER = {
-    inline_backend: 0,
-    'Qt5Agg': 2,
-    'QtAgg': 2,  # For Matplotlib 3.5+
-    'TkAgg': 3,
-    'MacOSX': 4,
+    inline_backend: "inline",
+    'Qt5Agg': 'qt5',
+    'QtAgg': 'qt5',  # For Matplotlib 3.5+
```

Comment:
```suggestion
    'QtAgg': 'qt',  # For Matplotlib 3.5+
```

I think this is the right value for this backend to match the one we're using in the `mapping` dict at the beginning of the `get_mpl_interactive_backend` method of `SpyderKernel` (otherwise I guess that function would fail to report the right interactive backend).

In addition, please check if this change is also necessary for the `Qt5Agg` backend (I guess it is but I'm not really sure).